docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md

```diff
@@ -0,0 +1,47 @@
+# Hosting on Windows
+
+## Hardware
+
+{% hint style="info" %}
+TODO: If you have experience in hosting Northstar servers and want to share your knowledge, please open a [pull request](https://github.com/R2Northstar/NorthstarWiki/pulls).
+{% endhint %}
+
+## Minimum requirements
+
+The current minimum requirements are as follows:
+
+**Requirements per server:**
+
+- 8GB free disk space (5GB of the game and 3 for Origin/EA app)
+- 3+ cores (2 might work, though)
+- 3GB+ total memory (RAM or swap will do)
+- Windows 8.1 or better (and built-in WARP [use `-softwared3d11`] or a graphics card)
```

Comment:
```suggestion
- Windows 8.1 or better (and built-in WARP) 
- A graphics card or the [`-software3d11` launch arg](./#dedicated-server-caveats)
```

unless this is supposed to be some markdown thing i just dont know yet. pretty sure this code works but github not letting me check :P